bin/README.md

```diff
@@ -0,0 +1,44 @@
+# Description
+This directory stores scripts for installing and setting up ATArchiver. The 
+directory contains two types of scripts - `setup_*` and `install_*`.
+
+## Install scripts
+Scripts prepended with `install` directive are installation scripts and they
+are used to install software and packages without regards to where they should
+be installed.
+
+## Setup scripts
+Setup scripts are prepended with `setup`. They are used to create users, 
+directories, setting up log files and permissions. Essentially, their role is 
+to make up everything that CTRL_IIP software needs.
+
+# Install
+* On ATArchiver machine, run as `root`
+```
+    $ source setup.env
```

Comment:
It might be a good idea to have a separate script as a single point to setup all of the version of the packages that will be installed as environmental variables for other downstream scripts to use.